scripts/gufi_common.py

```diff
@@ -257,6 +257,16 @@ def build_query(select, tables, where=None, group_by=None,
 
     return query
 
+# a helper function to show the gufi_query being executed. Helpful for debugging and education.
+def print_query(query_tokens):
+  formatted_string = ""
+  for index, token in enumerate(query_tokens):
+      if token.startswith('-') or index == len(query_tokens)-1:
+          formatted_string += '\n    ' + token
+      else:
+          formatted_string += ' ' + token
+  print('GUFI query is \n  {0}'.format(formatted_string))
```

Comment:
Would it make sense to write to to stderr instead of stdout so that this isn't mixed in with the output?

Reply:
I'm certainly happy to do that if you think best. But it might make sense to leave it to stdout since it isn't an error message. It will only show up when users explicitly ask for it so they should be prepared to handle it. And presumably they'll only turn it on during interactive sessions where it won't cause any problems with parsing or anything. Up to you. The other thing we might consider would be to do it on a single line. That would allow folks to filter it out with a grep if they want. But I sort of hate that because it's impossible to read if it's on a single line.  We could also add some sort of comment marker to it like:
# gufi query is
# gufi_query
#   -E 'flbah'
#   -n 224
Something like that.

LMK what you want please. Happy to do whatever but I think it's fine as is since folks will only have to deal with it when they explicitly ask for it.